tox.ini

```diff
@@ -17,9 +17,10 @@ commands = py.test \
 deps =
     flake8
     flake8-mutable
-commands = flake8
+commands = flake8 ocs_ci tests
 
 [flake8]
+basepython = python3
```

Comment:
So I'm really not sure about this; Just bringing it up for discussion;
I think it might be better to specifically state python3.6/python3.7, considering our code depends on Python>=3.6.
According to [the docs](https://tox.readthedocs.io/en/latest/config.html#conf-basepython), python3 simply calls.... python3, which I assume might be below version 3.6, and might cause possible issues with flake8.
If python3 is safe and shouldn't hinder the process in any way, feel free to disregard my comment :)

Reply:
Well, we need this to work on 3.6 *or* 3.7, so I'm not sure if we can express that properly here

Reply:
so what are the next steps here? drop the PR?

Reply:
I would vote for merging it since it addresses real issues. I just have no reviews...

Reply:
what about @MeridianExplorer comment? it will only pickup one version of python for CI tests?

Reply:
If you check the Travis build attached to this PR, you'll see it runs as normal

Reply:
thanks, yep dont see any issues.